### PR TITLE
Shard multiplexing docs

### PR DESCRIPTION
**Pull request by mDuo13**
Documents https://github.com/ripple/rippled/pull/3463/

### CODE REVIEW

**Review by ryangyoung — APPROVED** [review]
LGTM

**Review by undertome — APPROVED** [review]

File: content/tutorials/manage-the-rippled-server/configuration/configure-history-sharding.md [hunk 1]
```diff
@@ -6,36 +6,41 @@
 
 To configure your `rippled` to store shards of ledger history, complete the following steps:
 
-## 1. Determine how much space to allocate for the shard store
+## 1. Determine how much many shards to maintain
```
Comment (miguelportilla) [inline content/tutorials/manage-the-rippled-server/configuration/configure-history-sharding.md:9]: "Determine how much many shards to maintain" -> "Determine how many shards to maintain"

File: content/tutorials/manage-the-rippled-server/configuration/configure-history-sharding.md [hunk 2]
```diff
@@ -6,36 +6,41 @@
...
 
-Before you configure your `rippled` server to store history shards, you must decide how much disk space to allocate to the history shard store. This also affects how much history you keep in the default ledger store. You should consider the following when deciding what size to configure your shard store:
+Before you configure your `rippled` server to store history shards, you must decide how many history shards you want to keep, which is mostly determined by how much disk space have available for use in the shard store. This also affects how much history you keep in the default ledger store. You should consider the following when deciding what size to configure your shard store:
```
Comment (miguelportilla) [inline content/tutorials/manage-the-rippled-server/configuration/configure-history-sharding.md:11]: "disk space have available for use in the" -> "disk space is available for use in the"

**Comment by miguelportilla** [comment]
Left a couple of comments. Thanks for the improvement on this doc!

**Comment by mDuo13** [comment]
Thanks, @miguelportilla . Fixed those issues in 2c5d41dcef57eeaa3ba543aa63f10f2922667156

**Pull request merged**